security/ThreatModel.md

```diff
@@ -0,0 +1,8 @@
+One of the bigger security threats right now is phone number spoofing scams.

```

Comment:
elderly are always at risk ... but funny even that situation with a component of social hacking of clicking in the wrong place, downloading the wrong thing, or providing password or SS# details.  Almost always a combo